documentation/src/docs/getting-started.md

```diff
@@ -36,31 +36,30 @@ $ tar xfvz pravega-<version>.tgz
 
 This launches all the components of Pravega on your local machine.
 
-**Note:** This is for testing/demo purposes only, *do not* use this mode of deployment 
-in Production! 
+**Note:** This is for testing/demo purposes only, **do not** use this mode of deployment
+in Production!
 
 More options to run Pravega can be found in [Running Pravega](deployment/deployment.md) guide.
-           
 
 ```
 $ cd pravega-<version>
 $ bin/pravega-standalone
 ```
 
-By using the above commands, Pravega should be up and running very soon.
+By using the above commands, Pravega should be up and running immediately.
 
 More options and additional ways to run Pravega can be found in [Running Pravega](deployment/deployment.md).
 
-## Running a sample Pravega App
+## Running a sample Pravega Application
 
-It is very simple to run a sample pravega App. Pravega maintains a separate github repository for sample applications.  It is located at:
+It is very simple to run a sample Pravega Application. Pravega maintains a separate github repository for sample applications.  It is located at:
```

Comment:
I don't really like this style "It is very simple to...". Let's just be objective and say:

```
We have developed a few samples to introduce the developer to coding with Pravega here:

[Pravega Samples](https://github.com/pravega/pravega-samples)

```

Reply:
updated in the document